.github/workflows/greenplum-abi-tests.yml

```diff
@@ -102,27 +87,35 @@ jobs:
           fetch-depth: 0 # Specify '0' to fetch all history for all branches and tags.
           path: gpdb_src
 
-      - name: Install abi-dumper
+      - name: Install dependencies and abi-dumper
+        run: |
+          sudo gpdb_src/README.ubuntu.bash
+          sudo apt install -y libuv1-dev libpam0g-dev libldap-dev libipc-run-perl abi-dumper
+          sudo ln -fs python2 /usr/bin/python
```

Comment:
Is sudo required here?

Reply:
Yes, default `ubuntu-latest` container on GA doesn't run as root, so we need sudo to install packages.